test/FisherPrice_Alpaka/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+# This makes sure cmake knows where to find alpaka
+set(CMAKE_PREFIX_PATH "/install")
+# Then we must specify that we want to use alpaka
+find_package(alpaka REQUIRED)
```

```suggestion
find_package(alpaka)

if(NOT alpaka_FOUND)
  message(STATUS "No alpaka install found, disabling build of FisherPrice_Alpaka")
  return()
endif()
```

As alpaka won't be a hard dependency to begin with, the above is the simplest way to enable/disable as needed. We can add a proper option to enforce later.

---

I proposed a modified version of this in the previous comment.

Cheers,

Mark